src/universalSortSecondary.java

```diff
@@ -0,0 +1,232 @@
+package components.universalSort;
+
+/*
+ * Layered implementations of secondary methods for {@code universalSort}
+ */
+public abstract class universalSortSecondary implements universalSort {
+    /*
+     * Public members ---------------------------------------------------------
+     */
+
+    /*
+     * Tracks if this is sorted alphabetically
+     *
+     * @updates this
+     *
+     * @ensures {@code this} = true or {@code this} = false
+     */
+    @Override
+    public boolean isSortedAlpha = false;
```

Comment:
This is a bit strange syntax. I shouldn't see any fields in the abstract class. You will interact with the representation with kernel methods directly. Also, overriding fields seems weird. Ideally, there would be no docs because they would all be inherited.